README.asciidoc

```diff
@@ -17,23 +17,7 @@ on any other website without explicit permission.
 [float]
 == Requirements
 
-First, you need to make sure that you have the following installed:
```

Comment:
I'm just pointing folks right at docker by default because it *ought* to be simpler to set up.